doc/ansible/user-guide.md

```diff
@@ -264,15 +264,20 @@ $ docker push quay.io/example/memcached-operator:v0.0.1
 
 Kubernetes deployment manifests are generated in `deploy/operator.yaml`. The
 deployment image in this file needs to be modified from the placeholder
-`REPLACE_IMAGE` to the previous built image. To do this run:
+`REPLACE_IMAGE` to the previous built image.
+Also, we need to update the `imagePullPolicy` to `Always` (makes it easier for testing).
+
+To do the above run:
 ```
-$ sed -i 's|REPLACE_IMAGE|quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
+$ sed -i 's|image:.*|image: quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
+$ sed -i 's|imagePullPolicy:.*|imagePullPolicy: Always|g' deploy/operator.yaml
```

Comment:
Not required be added. See above that default value scaffolded will be Always.

Reply:
@camilamacedo86  - I think we still need to do the same, otherwise, how will these values be fixed before running this command `$ kubectl create -f deploy/operator.yaml` as suggested in the user [guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/user-guide.md#1-run-as-a-pod-inside-a-kubernetes-cluster).

I am new to this project, have I missed any steps?

Reply:
Ah, I see what you mean now? User guide with [v0.15.X](https://github.com/operator-framework/operator-sdk/blob/v0.15.x/doc/ansible/user-guide.md) is slightly different from that of the [master](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/user-guide.md).

Let me try it out quickly, will update the PR accordingly.

Reply:
Ah, those changes are already mentioned in the [v0.15.X](https://github.com/operator-framework/operator-sdk/blob/v0.15.x/doc/ansible/user-guide.md). The user guide in `master` branch is different.
I am updating my PR and will probably raise a PR against `v0.15.x` branch.